modules/ofiuco/0.3.7/presubmit.yml

```diff
@@ -0,0 +1,32 @@
+matrix:
+  platform:
+    - centos7
+    - debian10
+    - ubuntu2004
+    - macos
+  bazel:
+    - 7.x
```

Comment:
nit: Does this module support 6.x? If not, you can also put `bazel_compatibility = [">=7.0.0"]` in your MODULE.bazel file. See https://bazel.build/rules/lib/globals/module#parameters_5

Reply:
@meteorcloudy  added `bazel_compatibility = [">=7.0.0"]`

Reply:
Thanks!